### ISSUE TITLE
IE/Edge "No such interface supported"

### ISSUE THREAD

**Issue by kennysmoothx** [jwplayer/jwplayer#1925]
![Screenshot](http://i.imgur.com/vlhA65h.png?1)

### Expected Behavior

You should be able to use jwplayer().remove() cleanly with no errors.

### Actual Behavior

On IE/Edge latest versions, when you remove jwplayer using the .remove() method, it throws a "No such interface supported" error.

### Steps to reproduce

1. Setup JWPlayer()
2. Run jwplayer().remove()

### Environment

Windows IE/Edge

JWPLAYER 7.9 & 7.10

This is not an isolated issue with our pc/enviroment, this has been happening to hundreds of users accessing our production website with the latest player.

Debugging it even further, the error is coming from the `require-less/lessc.js` file. 

Specifically the ```createCSS()``` function.

And I think its this block

``` Line 7382
if (oldCss && keepOldCss === false) {
      oldCss.parentNode.removeChild(oldCss);
}
```

**Note:** This breaks any further action you wish to do with jwplayer, for instance if you wish to remove and setup jwplayer again, you can't. This breaks jwplayer.

Let me know if you have any questions

**Comment by jw-kaurand** [comment]
Hi @kennysmoothx! I was able to reproduce this issue you're referring to and we'll start looking at pulling this issue in to fix.  I was not able to reproduce it consistently though.  Can you share any information about the config you're using to set up the player?

**Comment by kennysmoothx** [comment]
Hello @jw-kaurand ,

It happens with any configuration I've tried, nothing in particular.

I posted the screenshot to shot it even happens with the jwplayer configuration from the dashboard.

In some incases you don't even have to remove the player, as soon as the player starts playing it throws the same error.

**Comment by robwalch** [comment]
Please provide a test page that reproduces the issue and we will reopen. I am unable to reproduce it without calling remove() twice which is not a bug.

**Comment by jienco** [comment]
I noticed that this issue happens only when including the cast configuration block, on Edge browser: 

`cast: { }`

Seems that the cast provider is throwing an exception, probably because of Edge not being capable of doing Chromecast/Airplay.

**Comment by robwalch** [comment]
OK. We have steps to reproduce with the following config:
```js
{
    "file": "//stream.mpd",
    "width": 680,
    "height": 400,
    "cast": {}
}
```
Scheduling this for v7.11.

**Comment by egreaves** [comment]
This was resolved by https://github.com/jwplayer/jwplayer-commercial/pull/3250 and released in `v7.10.5`.
